src/pudl/metadata/classes.py

```diff
@@ -288,6 +291,24 @@ def _check_max(cls, value, info: ValidationInfo):
                 raise ValueError("must be greater or equal to minimum")
         return value
 
+    def to_pandera_checks(self) -> list[pr.Check]:
```

Comment:
Eventually, it would be nice to have this be a proper `frictionless` field definition, and use the Pandera built-in conversion method.